manifest-build-tools/application/make_debian_packages.py

```diff
@@ -217,10 +218,20 @@ def write_downstream_parameter_file(build_directory, is_official_release, parame
         else:
             raise RuntimeError("Version of {0} is None. Maybe the repository doesn't contain debian directory ".format(rackhd_repo_dir))
 
+        manifest = Manifest(manifest_file)
+        rackhd_commit = ''
+        for repo in manifest.repositories:
+            if repo['repository'].endswith('RackHD') or repo['repository'].endswith('RackHD.git'):
```

Comment:
the git is case-insensitive for repository address, so endwith `rackhd` is also valid. You could first convert the repository address into lower case, then comparing the lower case string.